test/mixin/simple-entity-mixin-test.js

```diff
@@ -0,0 +1,110 @@
+import 'd2l-fetch/d2l-fetch.js';
+import { html, fixture, expect } from '@open-wc/testing';
```

Comment:
Experimenting with using `@open-wc/testing` framework as it seems simpler to use than the legacy polymer stuff.